docs/src/main/asciidoc/_developer-guide/ejb3/Jakarta_Enterprise_Beans_Database_Persistent_Timers.adoc

```diff
@@ -125,7 +125,7 @@ out of sync. Some application may find the `refresh-interval` configuration not
 need to programmatically refresh timers. This can be done with Jakarta Interceptors configured for those business methods
 that need this capability, as illustrated in the following steps:
 
-* Implement an Jakarta Interceptors that enables `wildfly.ejb.timer.refresh.enabled` to true. For example,
+* Implement a Jakarta Interceptors that enables `wildfly.ejb.timer.refresh.enabled` to true. For example,
```

Comment:
Your change is fine and I don't want to increase your scope of work, so feel free to ignore...

Either
"a Jakarta Interceptors interceptor"

or 

"an interceptor"

Reply:
Went with former akin to "Jakarta Enterprise Beans bean" :-)